src/tests/admin/conferenceRoomResourceAssociations/AResourceDeletedIsRemovedFromResourcesAssociatedInARoom.java

```diff
@@ -0,0 +1,77 @@
+package tests.admin.conferenceRoomResourceAssociations;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.common.UIMethods;
+import framework.pages.admin.HomeAdminPage;
+import framework.pages.admin.conferencerooms.RoomInfoPage;
+import framework.pages.admin.conferencerooms.RoomResourceAssociationsPage;
+import framework.pages.admin.conferencerooms.RoomsPage;
+import framework.pages.admin.resources.ResourcesPage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+import framework.utils.readers.JsonReader;
+
+/**
+ * TC01: Verify that a resource that is deleted it is removed from resources associated in a room
```

Comment:
Please change the title to:
Verify that a deleted resource is removed from resources associated to a room   or
Verify that a resource that is deleted is removed from  resources associated to a room